client/src/templates/Challenges/components/CompletionModalBody.js

```diff
@@ -0,0 +1,103 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import BezierEasing from 'bezier-easing';
+import GreenPass from '../../../assets/icons/GreenPass';
+
+const propTypes = {
+  blockName: PropTypes.string,
+  completedPercent: PropTypes.number
+};
+
+export class CompletionModalBody extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      animateProgress: false,
```

Comment:
```suggestion
```
I think this can go, since the `progress-bar-percent` can't be seen initially, so doesn't need hiding.